.ai/auto-review/github_reviewer.py

```diff
@@ -112,6 +133,22 @@ def post_line_comment(github: GitHub, file: str, text:str, line: int):
     except RepositoryError as e:
         Log.print_red("Failed line comment", e)
         return False
+    
+def post_line_comment_with_diff_hunks(github: GitHub, file: str, text:str, line: int, diff_hunk: str):
+    Log.print_green("Posting line", file, line, text)
+    try:
+        git_response = github.post_comment_to_line_with_diff_hunks(
+            text=text, 
```

Comment:
141 : Type error. The `diff_hunks` argument should be a list of strings, but `extract_diff_hunks` returns a list of strings. Pass the individual diff hunks to the function or modify the function signature.